pkg/mic/mic.go

```diff
@@ -410,24 +385,28 @@ func (c *Client) MakeAssignedIDs(azID *aadpodid.AzureIdentity, azBinding *aadpod
 			AvailableReplicas: 1,
 		},
 	}
+	// if we are in namespaced mode (or az identity is namespaced)
```

Comment:
comment seems redundant?

Reply:
Also do we need the if check? Would `azID.Namespace` be set to what we want regardless of it actually being namespaced?